src/lager.erl

```diff
@@ -486,6 +504,23 @@ safe_format(Fmt, Args, Limit, Options) ->
 safe_format_chop(Fmt, Args, Limit) ->
     safe_format(Fmt, Args, Limit, [{chomp, true}]).
 
+%% @private Print the format string `Fmt' with `Args' without a size limit.
+%% This is unsafe because the output of this function is unbounded.
+%%
+%% Log messages with unbounded size will kill your application dead as
+%% OTP mechanisms stuggle to cope with them.  So this function is
+%% intended <b>only</b> for messages which have a reasonable bounded
+%% size before they're formatted.
+%%
```

Comment:
I wonder if this is really a strong enough warning, and if it shouldn't have a README change as well - would be more likely to be seen there. Is there a set of README changes planned to go with this?